packages/block-editor/src/components/use-settings/README.md

```diff
@@ -1,8 +1,8 @@
-## Use Setting
+## Use Settings
 
-`useSetting` is a hook that will retrive the setting for the block instance that's in use. 
+`useSettings` is a hook that will retrieve the settings for the block instance that's in use.
```

Comment:
Should we keep a reference to the deprecated `useSetting()` in the README?

Reply:
Probably not, it's sufficiently documented in the jsdoc comments and the top-level README that's generated from them.